python/ray/data/context.py

```diff
@@ -259,6 +259,8 @@ def __init__(
         self.op_resource_reservation_enabled = DEFAULT_ENABLE_OP_RESOURCE_RESERVATION
         # The reservation ratio for ReservationOpResourceAllocator.
         self.op_resource_reservation_ratio = DEFAULT_OP_RESOURCE_RESERVATION_RATIO
+        # Minimum number of output blocks for a dataset.
+        self.min_num_blocks = DEFAULT_MIN_NUM_BLOCKS
```

Comment:
this name is a bit confusing here. something like "read_op_min_num_blocks" may be better.

Reply:
updated.